drivers/i2s/i2s_mcux_sai.c

```diff
@@ -998,13 +998,13 @@ static void audio_clock_settings(const struct device *dev)
 		LOG_ERR("wrong clock system configured");
 		return;
 	}
+	audioPllConfig.src = dev_cfg->pll_src;
 #endif
```

Comment:
The `#if CONFIG_CLOCK_CONTROL_MCUX_CCM_REV2` block should be moved out of the driver and into the soc.c file.

Reply:
so add a function in soc.c, right? let me add.